addons/mail/static/src/core/public_web/discuss_app_model.js

```diff
@@ -1,6 +1,9 @@
 import { Record } from "@mail/core/common/record";
 import { browser } from "@web/core/browser/browser";
 
+const NO_MEMBERS_DEFAULT_OPEN_LS = "mail.user_setting.no_members_default_open";
```

Comment:
we may want to have all the static local values in one file.

Reply:
The local storage entries have been initially added in discuss app model, but then chat hub and settings model also had a use.

I would keep their placement local, as to keep a better locality of concern for the time being. I'd make code cleaning in `master`, which would likely imply centralize definition of local storage fields probably?